src/picongpu/include/simulation_defines/unitless/visualization.unitless

```diff
@@ -40,18 +40,18 @@ namespace picongpu
 
     // setting 3: Plasma Wave
 #if( EM_FIELD_SCALE_CHANNEL1 == 3 || EM_FIELD_SCALE_CHANNEL2 == 3 || EM_FIELD_SCALE_CHANNEL3 == 3 )
-    PMACC_CASSERT_MSG( You_can_not_scale_your_preview_to_a_zero_plasma_density___change_visualization_param, ((GAS_DENSITY)>0.0 && gasProfile::GAS_ENABLED) );
+    PMACC_CASSERT_MSG( You_can_not_scale_your_preview_to_a_zero_plasma_density___change_visualization_param, ((BASE_DENSITY)>0.0 && densityProfile::GAS_ENABLED) );
```

Comment:
I can't find the definition of `GAS_ENABLED`, this looks like a BUG. 

Please remove GAS_ENABLED.